Cargo.toml

```diff
@@ -7,4 +7,7 @@ edition = "2021"
 
 [dependencies]
 yew = "0.19"
-jinko = { git = "https://github.com/jinko-core/jinko", default-features = false }
+#jinko = { git = "https://github.com/jinko-core/jinko", default-features = false }
+jinko = { path = "../jinko", default-features = false }
```

Comment:
I don't know if that change should be part of this PR :D

Reply:
Yup, should be fixed now

Reply:
Fixed